### DISCUSSION TITLE
MariaDB create user with REQUIRE SSL using the GUI

### DISCUSSION THREAD

**Discussion by dubidubno** [dbeaver/dbeaver#21036] [Q&A] [answered]
Is it possible tio create users in MariaDB with the REQUIRE SSL option without doing a manual command? I can't seem to find an option for it in the GUI.

It's documented here:
https://mariadb.com/kb/en/create-user/

**Comment by E1izabeth** [comment, marked as answer]
There is no such possibility. You can modify the final query manually in the SQL Editor to add REQUIRE SSL clause and execute it.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/cc6777c0-8471-4ff2-90b4-753f0ff57d80)
